### ISSUE TITLE
[IntelliJ-ThirdParty][A11yMAS][Accessibility][Keyboard][MAS2.1.1][IntelliJ][Wipro]: Unable to access "Context menu" with keyboard for show properties dialog.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1515]
**Repro Steps:**

1. In ""Azure Explorer"" , select the created app under web apps.
2. Try to access the "Context menu" using  "Shift + F10" with keyboard.

**Expected Result:**
User should be able to access the "Context menu" with keyboard.

**Actual Result:**
Unable to access "Context menu" with keyboard for show properties dialog.

**Platform:**
IntelliJ IDEA 2017.3.4 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
**OSBuild Version:** Version 1709(Os Build 16299.192)

![context menu is not accessible with keyboard in show properties](https://user-images.githubusercontent.com/22956130/36150860-01431546-107a-11e8-8dbe-af058cf85793.png)

**Comment by Eskibear** [comment]
duplicated with #1519

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749980
